guides/user/canary/config/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: single
+title:  "Configure Canary for an application"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Before you can add a canary stage to a pipeline, you need to configure what the
+canary consists of, including...
+
+* The specific metrics to evaluate, and a logical grouping of those metrics
+* Default scoring thresholds (which can be overridden in the
+canary stage)
+* Optionally, one or more filter templates
+* A name by which a canary stage can choose this config
+
+Canary configuration is done per Spinnaker
+[application](/concepts/#applications). For each
+application set up to support canary, you create one or more configs. Your
+setup might even be that all configurations are available to all
+applications.
+
+The configuration you create here is available to canary stages in pipelines,
+but those [stages are defined separately](guides/user/canary/stage/).
+
+<!---
+TODO: figure out why this is done and how it's used
+--->
+Separately, you can [group individual metrics by Stackdriver
+resource types](#group_the_metrics).
+
+## Prerequisites
+
+By default, canary is not enabled for new applications. Several things need to
+happen before you see the __Canary__ tab in Deck:
+
+* The person setting up Spinnaker for you must [set up Canary](/setup/canary/).
+
+  This includes specifying whether all canary configs are available to all
```

Comment:
Turns out I had already added the setting for the flag to halyard. This PR flips the default setting from `false` to `true`: https://github.com/spinnaker/halyard/pull/888

We should update the text here and everywhere to reflect the change.